lib/react/server_rendering/webpacker_manifest_container.rb

```diff
@@ -21,11 +21,23 @@ def find_asset(logical_path)
           dev_server_asset
         else
           # Read the already-compiled pack:
-          full_path = Webpacker::Manifest.lookup_path(logical_path).to_s
+          full_path = file_path(logical_path).to_s
           File.read(full_path)
         end
       end
 
+      def manifest
+        Webpacker.respond_to?(:manifest) ? Webpacker.manifest : Webpacker::Manifest
+      end
+
+      def file_path path
+        manifest.respond_to?(:lookup_path) ? manifest.lookup_path(path) : File.join(config.output_path, manifest.lookup(path).split('/')[2..-1])
+      end
+
+      def config
+        !!defined?(Webpacker::Configuration) ? Webpacker::Configuration : Webpacker.config
```

Comment:
I think this needs to be

```rb
def config
  Webpacker.respond_to?(:config) ? Webpacker.config : Webpacker::Configuration
end
```

Because even in Webpacker 3, the `Webpacker::Configuration` constant is defined.